doc/cprover-manual/jbmc-user-manual.md

```diff
@@ -0,0 +1,229 @@
+\ingroup module_hidden
+\page jbmc-user-manual JBMC User Manual
+
+
+\section jbmc-manual-intro Introduction
+
+JBMC is a bounded model checking tool for the Java language. It follows the
+same principles as CBMC, thus the existing
+[documentation for CBMC](http://www.cprover.org/cprover-manual/cbmc.html)
+largely applies to JBMC.
+
+In brief, JBMC creates a model from Java Bytecode (given as input as .class
+files) and performs static analysis and symbolic simulation which allow to
+prove or refute properties. These properties are either automatically
+generated, or provided by the user in the form of assertions.
+
+Using examples, the following section will guide you through the basic
+features of JBMC, and explore the most useful features.
+
+
+\section jbmc-tutorial JBMC tutorial
+
+\subsection jbmc-manual-auto-assert Automatic assertions
+
+Consider the following simplistic `Example.java` file which we have compiled
+into `Example.class` and which contains a utility method to query a given
+array at a given index:
+```java
+1  package core;
+2
+3  public class Example {
+4
+5      public static int query(int[] array, int index) {
+6          return array[index];
+7      }
+8  }
+```
+
+Now let's run the following command to let JBMC tell us about potential errors
+in our `query` method.
+```
+jbmc Example.class --function core.Example.query
```

Comment:
That's a bit strange that you `cd` first into `core` and then run `jbmc` from there. I'd say `jbmc core/Example.class ...`. 
Side note: At some point that will actually change to running `jbmc core.Example ...` (akin to `java core.Example`).

Reply:
Is `core` a good name? What about `tutorial` or `examples`?

Reply:
Actually, about your first comment, I'm not cd-ing into anything. There is no `core` folder. I compiled the java file using `javac`, and it's not within a maven project, so the folder hierarchy doesn't follow the packages. Is that confusing? Should I remove the package from the examples?